[2024-01-26T13:15:03.774-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T16:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T13:15:03.816-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T16:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T13:15:03.833-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T16:00:00+00:00 [queued]>
[2024-01-26T13:15:03.843-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T16:00:00+00:00 [queued]>
[2024-01-26T13:15:03.844-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-26T13:15:03.869-0300] {taskinstance.py:1382} INFO - Executing <Task(LambdaInvokeFunctionOperator): invoke_lambda_function> on 2024-01-26 16:00:00+00:00
[2024-01-26T13:15:03.871-0300] {standard_task_runner.py:57} INFO - Started process 3602 to run task
[2024-01-26T13:15:03.873-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'how-desafio-03', 'invoke_lambda_function', 'scheduled__2024-01-26T16:00:00+00:00', '--job-id', '223', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpvrzu538d']
[2024-01-26T13:15:03.874-0300] {standard_task_runner.py:85} INFO - Job 223: Subtask invoke_lambda_function
[2024-01-26T13:15:03.921-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T16:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T13:15:03.922-0300] {task_command.py:416} INFO - Running <TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T16:00:00+00:00 [running]> on host 14111-NB.
[2024-01-26T13:15:04.006-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='how-desafio-03' AIRFLOW_CTX_TASK_ID='invoke_lambda_function' AIRFLOW_CTX_EXECUTION_DATE='2024-01-26T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-26T16:00:00+00:00'
[2024-01-26T13:15:04.006-0300] {lambda_function.py:211} INFO - Invoking AWS Lambda function: how-desafio-03 with payload: {"date": "1706285700"}
[2024-01-26T13:15:04.014-0300] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-01-26T13:15:04.014-0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-01-26T13:15:04.946-0300] {lambda_function.py:220} INFO - Lambda response metadata: {'RequestId': 'f19718a1-6a9e-4e9b-8fe1-6a41f46f8a2e', 'HTTPStatusCode': 202, 'HTTPHeaders': {'date': 'Fri, 26 Jan 2024 16:15:26 GMT', 'content-length': '0', 'connection': 'keep-alive', 'x-amzn-requestid': 'f19718a1-6a9e-4e9b-8fe1-6a41f46f8a2e', 'x-amzn-remapped-content-length': '0', 'x-amzn-trace-id': 'root=1-65b3da9e-1cb7b04e09c7c8ed23a2c9c1;sampled=0'}, 'RetryAttempts': 0}
[2024-01-26T13:15:04.947-0300] {lambda_function.py:230} INFO - Lambda function invocation succeeded: {'RequestId': 'f19718a1-6a9e-4e9b-8fe1-6a41f46f8a2e', 'HTTPStatusCode': 202, 'HTTPHeaders': {'date': 'Fri, 26 Jan 2024 16:15:26 GMT', 'content-length': '0', 'connection': 'keep-alive', 'x-amzn-requestid': 'f19718a1-6a9e-4e9b-8fe1-6a41f46f8a2e', 'x-amzn-remapped-content-length': '0', 'x-amzn-trace-id': 'root=1-65b3da9e-1cb7b04e09c7c8ed23a2c9c1;sampled=0'}, 'RetryAttempts': 0}
[2024-01-26T13:15:05.010-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=how-desafio-03, task_id=invoke_lambda_function, execution_date=20240126T160000, start_date=20240126T161503, end_date=20240126T161505
[2024-01-26T13:15:05.053-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-26T13:15:05.081-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
